server/protocols/webadmin/webadmin-data/pom.xml

```diff
@@ -97,6 +100,13 @@
         <dependency>
             <groupId>org.junit.jupiter</groupId>
             <artifactId>junit-jupiter-engine</artifactId>
+            <version>${junit-jupiter-engine.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-params</artifactId>
+            <version>${junit-jupiter-engine.version}</version>
```

Comment:
Include this dependency in james-project's pom.xml (we tend to avoid versions in children's POM to make librairy upgrades easier.)